src/components/SearchResult/SearchResult.tsx

```diff
@@ -0,0 +1,45 @@
+import styles from './SearchResult.module.css';
+
+export type SearchResultProps = {
+  poster_path: object;
+  name: string;
+  vote_average: number;
```

Comment:
The format surprises me: Wouldn't it be better to either use PascalCase or snake_case instead of both?